runtime/near-vm-runner-standalone/src/main.rs

```diff
@@ -231,5 +276,72 @@ fn main() {
             state: State(fake_external.fake_trie),
         })
         .unwrap()
-    )
+    );
+
+    if do_profile {
+        let profile_data = profile_data.borrow();
+
+        let mut host_gas = 0u64;
+        for e in 0..ExtCosts::count() {
+            host_gas += profile_data[e as usize];
+        }
+
+        let mut action_gas = 0u64;
+        for e in 0..Actions::count() {
+            action_gas += profile_data[e as usize + ExtCosts::count()];
+        }
+
+        let wasm_gas = profile_data[Actions::count() + ExtCosts::count()];
```

Comment:
After some thought it might be good if we had the option to write to JSON file.

Reply:
Well, maybe, but not sure what usecase would be.

Reply:
Well currently in near-sdk-as, it uses the rust binary and expects the output to be the JSON of the outcome.  It is possible to work around this, but it would be a lot simpler if the only output is the outcome.